src/commands/publish_images.yml

```diff
@@ -18,6 +18,10 @@ parameters:
     description: Path to Dockerfile
     type: string
     default: ""
+  dockerbuild_context:
+    description: Path to Dockerbuild context
+    type: string
+    default: "./"
```

Comment:
to be consistent with `dockerfile_path` and `container_image_name`, the default should be `""` too.